docs/server-request.md

```diff
@@ -22,13 +22,36 @@ A low-level action that allows for any kind of HTTP request to be performed.
 - [Parse JSON](/docs/Actions/CallApi.md) - creates variables from JSON returned by API calls
 - [Regex](/docs/Actions/CallApi.md) - create variables by matching patterns inside the content
 
-## `Parameter Reference`
+## `Input Parameter Reference`
 
 | Parameter     | Description                           | Supports Variables |
 |---------------|---------------------------------------|--------------------|
-| URL           | Absolute URL where the resource to be invoked lives. | Yes |
-| HTTP Method   | The operations to perform against the URL.<br/> Possible values are GET, POST, PUT, DELETE, HEAD. | No |
+| URL           | The URL where the resource to be invoked lives. It can be relative or absolute. | Yes |
+| Enforce SSL   | This will for the request to be sent to a secure version of the URL(https instead of http). This is useful for the instances in which the URL comes from a variable and you want to ensure it is secured.| No |
+| Timeout       | The amount of time in seconds in which the response must come or the request fails. If nothing is provided, it will default to 100 seconds. If you don't want to wait for the request to finish(when you must leverage higher Timeout values) we suggest using [Execute Actions Async](/docs/Actions/ExecuteAsync.md). | Yes |
+| HTTP Method   | The operations to perform against the URL.<br/> Possible values are GET, POST, PUT, DELETE, HEAD, PATCH. | No |
 | Data          | If the HTTP Method is POST or PUT, data is allowed to be sent to the target URL. The data can take various forms, from key-value pairs to complex JSON. For the target URL to understand the format, it needs to be passed through the Content-Type header. | Yes |
+|Do not Escape Tokens in Data | By default, we escape tokens in order to not break the XML structure. If you need your tokens to bring XML data, you will have to check this. | No|
+| Disable Refer Header | By default, we append a referer header  but some APIs will throw errors if this header is sent. If the resource  you want to use doesn't need the referer header, check this. | No |
+| Headers | Additional headers to pass with the request. | Yes |
+| Use DNN Proxy Settings | Check this if you want to use the proxy settings you have set for your DNN instance. | No |
+| Add Current Cookies | This will add the current cookies of the session to the request. | No |
+| Cookie Container Token |  This will name a container in which to store the cookies. If you want to use cookies from the previous actions or pass them along the execution stack, make sure they have same name. | No |
+| Url Token Context | If you are using variables to provide the  URL this will decide whether the variables will be URL encoded or not. | No |
```

Comment:
If I understand correctly, this option only affects the URL. If this is the case, let's move it directly after the URL parameter both in the documentation and in the action config.

Reply:
Also, let's rename from Url Token Context to URL Token Encoding and the 2 options to URL Encode and No Encoding, both in documentation and config. 
Also, let's explain what encoding means.